Build-your-first-dApp.md

```diff
@@ -308,25 +309,25 @@ const MoodContractABI = [
 ]
 ```
 
-3. Next, Define an ethers provider. In our case it is Goerli:
+3. Next, Define an ethers provider. In our case it is "any":
+  - This will allow your dApp to be compatible with both, Sepolia and Goerli networks
```

Comment:
This will allow any network right? Not just Sepolia and Goerli. I wonder if we'll get an instant where someone accidentally does this with Metamask on Ethereum and uses real tokens.

Reply:
![image](https://user-images.githubusercontent.com/106323731/225731778-a97dcf04-67a8-488f-84ce-afeb19411911.png)

I haven't tested if it works on the mainnet, but we are clearly saying that the contract should be deployed on the Sepolia network + we advise using a wallet without funds.

When I try to specify "sepolia" instead of any it says it is an invalid argument. 
It doesn't show this error for "goerli" and "mainnet", so this way it should work with any EVM network that MM is connected to.

Reply:
I'm a little worried since this is in the freshman track. Maybe we can emphasize more on the Testnet connection on MM and add a warning say if the user is connected to mainnets it will use real ETH.

Reply:
I agree that `any` feels dangerous over here. According to ethers docs, `sepolia` should work fine, is it not working at all?

Reply:
![image](https://user-images.githubusercontent.com/106323731/226372656-a8a59558-b45f-4472-8ad5-941ff66358a6.png)

passing ``sepolia`` into Web3Provider instead of ``goerli`` results in invalid argument error. 
I wasn't able to find another way of using Sepolia for this, because docs are the way they are... not very helpful

Reply:
> ![image](https://user-images.githubusercontent.com/106323731/226372656-a8a59558-b45f-4472-8ad5-941ff66358a6.png)
> 
> passing `sepolia` into Web3Provider instead of `goerli` results in invalid argument error. I wasn't able to find another way of using Sepolia for this, because docs are the way they are... not very helpful

I faced this error while going through this lesson yesterday.
A fix for this is to use:
`<script src="https://cdn.ethers.io/lib/ethers-5.7.2.umd.min.js" type="application/javascript"></script>`